ksql-engine/pom.xml

```diff
@@ -144,6 +144,16 @@
             <scope>test</scope>
         </dependency>
 
+        <!-- Transitive dependency of wiremock-jre8 that's excluded in ksql-parent pom.xml
+             because it brings an older version of jetty than we'd like -->
+        <dependency>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-webapp</artifactId>
+            <!-- jetty.version definition comes from rest-utils -->
+            <version>${jetty.version}</version>
+            <scope>test</scope>
+        </dependency>
+
```

Comment:
This seems unrelated to the Base64 change. I take it the build was already failing, and this was needed to make it pass? 
I am wondering whether this should be a separate pull request. I'll let reviewers from KSQL side make the call. (cc @agavra )

Reply:
it's fine to keep it in this PR if the build is otherwise broken. affects test dependencies only so if the build is green I'm happy with it.